packages/fxa-settings/src/lib/types.ts

```diff
@@ -5,3 +5,11 @@
 export type LinkType = 'reset-password' | 'signin';
 
 export type LinkStatus = 'valid' | 'used' | 'expired' | 'damaged';
+
+export enum RelyingParties {
```

Comment:
Can we call this `ServiceNames` instead? I think we also want `Default` to `account settings` as well. I don't think we want `FirefoxAccounts` here either.

Reply:
I'd lean towards leaving in "Firefox" or "Mozilla" since that's how they're referenced elsewhere, so I think it's easier to cross reference! I'll add the comment in and rename!